python/ray/services.py

```diff
@@ -789,3 +814,21 @@ def start_ray_head(address_info=None,
                              start_workers_from_local_scheduler=start_workers_from_local_scheduler,
                              num_cpus=num_cpus,
                              num_gpus=num_gpus)
```

Comment:
The methods `start_ray_processes`, `start_ray_head`, and `start_ray_node` should probably get rid of the `redirect_output` argument and replace it with `redirect_stdout` and `redirect_stderr`, and pipe it all the way through, right?

Also, is it possible to redirect to `/dev/null` as a special case of this functionality?